pkgs/development/python-modules/oncalendar/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, python
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "oncalendar";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "cuu508";
+    repo = "oncalendar";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-eQYxOnL4/TJbY/nijcPl8TqK2MrwcdISKGfZQoI7828=";
+  };
+
+  checkPhase = ''
+    ${python.interpreter} setup.py test
```

Comment:
setup.py test is deprecated. Since this is just unittest, we can probably execute them with pytest without problems

Reply:
It's used by upstream and as even eggs are still supported by most python tooling I'd be okay with taking the risk of having to change it in an update as a maintainer ;D 

But anyways, uses pytestCheckHook now. The unittest one would need extra flags, so I kept it simple.

Hope we can get this merged before the 4th minor release during this PRs lifetime :)